From 718904eb504d765162edd14027730356e1a53fd5 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Mon, 25 Apr 2022 12:26:04 +0200
Subject: [PATCH 0413/1695] plat-stm32mp2: conf: support IPCC driver

Default enable IPCC driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9ebe64ba8d457f4034dc12b9ccd5c8f34006e62a plat-stm32mp2: conf: support IPCC driver]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: Id9ecf689d2e2baf3a7cddc169a9b286c6c53ee37
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/249371
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 0b09787f5..c9f8ba692 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -73,6 +73,7 @@ CFG_STM32_GPIO ?= y
 CFG_STM32_HPDMA ?= y
 CFG_STM32_HSEM ?= y
 CFG_STM32_IAC ?= y
+CFG_STM32_IPCC ?= y
 CFG_STM32_OMM ?= y
 CFG_STM32_RIF ?= y
 CFG_STM32_RIFSC ?= y
-- 
2.47.2

